prime-router/src/main/kotlin/azure/SettingsFacade.kt

```diff
@@ -285,8 +285,8 @@ class SettingsFacade(
 
             db.insertDeletedSettingAndChildren(current.settingId, claims.userName, OffsetDateTime.now(), txn)
             db.deactivateSettingAndChildren(current.settingId, txn)
-
-            Pair(AccessResult.SUCCESS, "")
+            // returned content-type is json/application, so return empty json not empty string
+            Pair(AccessResult.SUCCESS, "{}")
```

Comment:
I'm not sure Delete was even working in staging? Maybe something changed in the client library, but if the content-type returned is json, then it always try to JSON.parse() it... even if it's empty. The best fix is to just return empty json.